docs/deploy/docker.md

```diff
@@ -27,6 +27,7 @@ services:
     volumes:
       - umap_userdata:/srv/umap/uploads
       # FIX the path on the left, below, to your location 
+      # OPTIONAL, you can comment the line below out
```

Comment:
Maybe explain why it's optional, and what would happen if you don't specify it (default values will be used, and then ENV variables will be read)

Reply:
tried to add more context while keeping the line short. My personal problem with that setting was that while using docker there was an error trying to read the file, if file did not exist then a folder called `umap.conf` was created. Removing the setting was enough for me to make the container run flawlessly